Git-commit: b93f9499427cf15ee39877c43052ed26527e59ff
From: Linus Torvalds <torvalds@linux-foundation.org>
Date: Mon, 26 Apr 2021 10:48:07 -0700
Subject: [PATCH] certs: add 'x509_revocation_list' to gitignore
References: bsn#19
Patch-mainline: v5.10.52

commit 81f202315856edb75a371f3376aa3a47543c16f0 upstream.

Commit d1f044103dad ("certs: Add ability to preload revocation certs")
created a new generated file for revocation certs, but didn't tell git
to ignore it.  Thus causing unnecessary "git status" noise after a
kernel build with CONFIG_SYSTEM_REVOCATION_LIST enabled.

Add the proper gitignore magic.

Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Kai Liu <kai.liu@suse.com>
---
 certs/.gitignore | 1 +
 1 file changed, 1 insertion(+)

diff --git a/certs/.gitignore b/certs/.gitignore
index 2a2483990686..6cbd1f1a5837 100644
--- a/certs/.gitignore
+++ b/certs/.gitignore
@@ -1,2 +1,3 @@
 # SPDX-License-Identifier: GPL-2.0-only
 x509_certificate_list
+x509_revocation_list
-- 
2.31.1

